spring-cloud-services-spring-connector/src/main/java/io/pivotal/spring/cloud/service/config/PlainTextOAuth2ConfigClient.java

```diff
@@ -75,8 +75,9 @@ public Resource getConfigFile(String profile, String label, String path) {
 				"Spring application name is undefined.");
 
 		Assert.isTrue(
-				configClientProperties.getUri() != null
-						&& !configClientProperties.getUri().isEmpty(),
+				configClientProperties.getUri().length > 0 &&
```

Comment:
Consider using `Assert.notEmpty()` and/or `Assert.hasText()` instead of the three checks in the isTrue block. Otherwise, nothing wrong with this. Something like this may be a little cleaner:

```
Assert.notEmpty(configClientProperties.getUri(), "Config server URI is undefined");
Assert.hasText(configClientProperties.getUri()[0], "Config server URI is undefined.");
```